src/components/MessageSection.jsx

```diff
@@ -9,31 +9,29 @@ import messaging from "../images/conversation.svg";
 
 function MessageSection() {
   return (
-    <div className="message-section">
```

Comment:
why was div `message-section` removed?

Reply:
unnecessary container. Moved styles to `message-container` and had the same effect

Reply:
changes approved.